### PR TITLE
Music in Tavern

### PR DESCRIPTION
**Pull request by B0ZIA**

### CODE REVIEW

File: Scenes/UserInterface/UIContainer.gd [hunk 1]
```diff
@@ -48,6 +50,7 @@ func _on_RevivalEnterBtn_pressed():
 
 func _on_TavernEnterBtn_pressed():
 	revival_enter_btn.set_pressed(false)
+	music_manager.play_music(music_manager.Musics.TAVERN_MUSIC)
```
Comment (SLASHLogin) [inline Scenes/UserInterface/UIContainer.gd:53]: Player can enter the tavern by dying, what means that player won't hear proper track. Please use this method in other place or copy it so it includes the other case

Comment (SLASHLogin) [reply Scenes/UserInterface/UIContainer.gd:53]: Another things is that players can click this button and music starts playing before they enter the tavern

Comment (B0ZIA) [reply Scenes/UserInterface/UIContainer.gd:53]: Ok, I'm working on it, I just added a manager and plugged in a few places ;)

File: Scripts/MusicManager.gd [hunk 2]
```diff
@@ -0,0 +1,26 @@
+extends Node2D
```
Comment (Ajver) [inline Scripts/MusicManager.gd:1]: MusicManager doesn't need to be `Node2D`. You can change it to simple `Node`.

File: Scripts/MusicManager.gd [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+			$MusicPlayer.stream = tavern_music
+		Musics.GAMEPLAY_MUSIC:
+			$MusicPlayer.stream = gameplay_music
+	$MusicPlayer.play()
+
+enum Musics {
+	MAIN_MENU_MUSIC = 1,
+	TAVERN_MUSIC = 2,
+	GAMEPLAY_MUSIC = 3
+}
```
Comment (Ajver) [inline Scripts/MusicManager.gd:26]: Enums are more readable while they are at the top of the document. I recommend to move it to line 7 or so.

File: Scenes/World/World.tscn [hunk 4]
```diff
@@ -59,12 +61,11 @@ layer = 62
...
-replay_delay = 5.0
-
 [node name="GameOverSound" type="AudioStreamPlayer" parent="."]
 stream = ExtResource( 15 )
 volume_db = -5.0
+
+[node name="MusicManager" parent="." instance=ExtResource( 16 )]
+main_menu_music = ExtResource( 17 )
+tavern_music = ExtResource( 13 )
+gameplay_music = ExtResource( 14 )
```
Comment (Ajver) [inline Scenes/World/World.tscn:71]: I think, better make MusicManager autoloaded singleton. The music now isn't playing in the main menu screen, what is a bug. And it should be accessible in other scenes as well.

**Comment by dyedmagenta** [comment]
@Ajver @B0ZIA @SLASHLogin
I've finished this task, you can review it now.

**Review by SLASHLogin — APPROVED** [review]

**Pull request merged**